### PR TITLE
"Buzz" intake on ready

### PR DESCRIPTION
**Pull request by vincentw921**
plz accept

### CODE REVIEW

File: src/main/java/com/stuypulse/robot/RobotContainer.java [hunk 1]
```diff
@@ -191,7 +191,10 @@ private void configureOperatorBindings() {
             .whileTrue(
                 new LEDSet(LEDColor.RED)
                     .andThen(new ManagerValidateState())
-                    .andThen(new ArmReady()));
+                    .andThen(new ArmReady())
+                    .alongWith(new IntakeAcquire()).withTimeout(0.2)
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/RobotContainer.java:195]: it looks like this timeout went onto the whole commanf group, when it should just be only on the intake acquire, ALSO you need to stop the intake after this wait command.

so maybe it should be a group of 
intake acquire, wait 0.2 seconds, intake stop

**Review by selym3 — CHANGES_REQUESTED** [review]

**Comment by selym3** [comment]
Also I wonder if you can cancel the whole operation if it stalls, but with such a low time it takes in it's probably fine

File: src/main/java/com/stuypulse/robot/RobotContainer.java [hunk 2]
```diff
@@ -188,10 +188,14 @@ private void configureOperatorBindings() {
 
         // ready & score
         operator.getLeftBumper()
+            .onTrue(new IntakeAcquire().withTimeout(0.2))
```
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/RobotContainer.java:191]: The intake never stops. You need an IntakeStop after IntakeAcquire ends.

**Review by anivanchen — CHANGES_REQUESTED** [review]

File: src/main/java/com/stuypulse/robot/RobotContainer.java [hunk 3]
```diff
@@ -188,11 +190,15 @@ private void configureOperatorBindings() {
 
         // ready & score
         operator.getLeftBumper()
+            .onTrue(new IntakeAcquire()
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/RobotContainer.java:193]: whileTrue so it ends when the button is let go of

**Review by selym3 — CHANGES_REQUESTED** [review]

**Review by BenG49 — APPROVED** [review]

**Pull request merged**